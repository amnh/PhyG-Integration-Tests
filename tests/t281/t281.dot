digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    31 [label=HTU31];
    32 [label=HTU32];
    33 [label=HTU33];
    34 [label=HTU34];
    35 [label=HTU35];
    36 [label=HTU36];
    16 -> 0 [label=83.5];
    16 -> 23 [label=91.5];
    17 -> 9 [label=147.0];
    17 -> 18 [label=76.5];
    18 -> 7 [label=158.0];
    18 -> 19 [label=68.5];
    19 -> 6 [label=164.5];
    19 -> 22 [label=56.0];
    20 -> 17 [label=57.0];
    20 -> 32 [label=1026.5];
    21 -> 12 [label=195.0];
    21 -> 33 [label=1016.0];
    22 -> 2 [label=164.0];
    22 -> 21 [label=66.0];
    23 -> 20 [label=26.0];
    23 -> 24 [label=45.0];
    24 -> 3 [label=166.5];
    24 -> 28 [label=48.0];
    25 -> 27 [label=39.0];
    25 -> 35 [label=68.5];
    26 -> 10 [label=109.5];
    26 -> 32 [label=988.0];
    27 -> 5 [label=196.5];
    27 -> 34 [label=1127.0];
    28 -> 31 [label=58.0];
    28 -> 36 [label=56.0];
    29 -> 14 [label=140.5];
    29 -> 33 [label=1039.5];
    30 -> 26 [label=71.0];
    30 -> 34 [label=1134.5];
    31 -> 11 [label=157.0];
    31 -> 29 [label=69.0];
    32 -> 1 [label=1050.0];
    33 -> 4 [label=1108.0];
    34 -> 15 [label=1197.0];
    35 -> 8 [label=192.5];
    35 -> 13 [label=157.5];
    36 -> 25 [label=43.0];
    36 -> 30 [label=34.0];
}
//3242.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=87.5];
    16 -> 23 [label=94.5];
    17 -> 9 [label=145.5];
    17 -> 18 [label=79.0];
    18 -> 7 [label=157.5];
    18 -> 19 [label=68.5];
    19 -> 6 [label=163.5];
    19 -> 22 [label=57.0];
    20 -> 1 [label=132.5];
    20 -> 17 [label=57.0];
    21 -> 4 [label=173.0];
    21 -> 12 [label=193.0];
    22 -> 2 [label=160.5];
    22 -> 21 [label=67.0];
    23 -> 20 [label=23.5];
    23 -> 24 [label=41.0];
    24 -> 3 [label=164.0];
    24 -> 26 [label=45.5];
    25 -> 5 [label=179.5];
    25 -> 29 [label=69.0];
    26 -> 28 [label=63.0];
    26 -> 30 [label=40.5];
    27 -> 10 [label=145.0];
    27 -> 15 [label=175.0];
    28 -> 11 [label=162.5];
    28 -> 14 [label=150.5];
    29 -> 8 [label=194.5];
    29 -> 13 [label=162.5];
    30 -> 25 [label=56.5];
    30 -> 27 [label=49.0];
}
//2887.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=87.5];
    16 -> 23 [label=94.5];
    17 -> 9 [label=145.5];
    17 -> 18 [label=79.0];
    18 -> 7 [label=157.5];
    18 -> 19 [label=68.5];
    19 -> 6 [label=163.5];
    19 -> 22 [label=57.0];
    20 -> 1 [label=132.5];
    20 -> 17 [label=57.0];
    21 -> 4 [label=173.0];
    21 -> 12 [label=193.0];
    22 -> 2 [label=160.5];
    22 -> 21 [label=67.0];
    23 -> 20 [label=23.5];
    23 -> 24 [label=41.0];
    24 -> 3 [label=164.0];
    24 -> 26 [label=45.5];
    25 -> 5 [label=179.5];
    25 -> 29 [label=69.0];
    26 -> 28 [label=63.0];
    26 -> 30 [label=40.5];
    27 -> 10 [label=145.0];
    27 -> 15 [label=175.0];
    28 -> 11 [label=162.5];
    28 -> 14 [label=150.5];
    29 -> 8 [label=194.5];
    29 -> 13 [label=162.5];
    30 -> 25 [label=56.5];
    30 -> 27 [label=49.0];
}
//2887.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=82.5];
    16 -> 22 [label=84.5];
    17 -> 9 [label=146.5];
    17 -> 18 [label=73.0];
    18 -> 7 [label=162.5];
    18 -> 19 [label=61.5];
    19 -> 6 [label=168.5];
    19 -> 21 [label=61.0];
    20 -> 1 [label=139.5];
    20 -> 17 [label=50.0];
    21 -> 2 [label=179.0];
    21 -> 12 [label=208.0];
    22 -> 20 [label=25.5];
    22 -> 23 [label=41.0];
    23 -> 3 [label=160.0];
    23 -> 25 [label=47.5];
    24 -> 5 [label=180.5];
    24 -> 29 [label=71.0];
    25 -> 28 [label=61.5];
    25 -> 30 [label=45.5];
    26 -> 4 [label=137.0];
    26 -> 14 [label=135.0];
    27 -> 10 [label=137.5];
    27 -> 15 [label=173.5];
    28 -> 11 [label=149.0];
    28 -> 26 [label=65.5];
    29 -> 8 [label=196.0];
    29 -> 13 [label=158.0];
    30 -> 24 [label=57.0];
    30 -> 27 [label=49.5];
}
//2853.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=84.0];
    16 -> 22 [label=89.0];
    17 -> 9 [label=149.5];
    17 -> 18 [label=71.5];
    18 -> 7 [label=166.5];
    18 -> 19 [label=62.0];
    19 -> 6 [label=170.5];
    19 -> 21 [label=60.0];
    20 -> 1 [label=135.0];
    20 -> 17 [label=52.0];
    21 -> 2 [label=181.0];
    21 -> 12 [label=209.0];
    22 -> 20 [label=24.5];
    22 -> 23 [label=39.0];
    23 -> 3 [label=163.5];
    23 -> 26 [label=50.5];
    24 -> 25 [label=67.5];
    24 -> 29 [label=62.0];
    25 -> 5 [label=187.0];
    25 -> 15 [label=152.0];
    26 -> 28 [label=59.5];
    26 -> 30 [label=41.0];
    27 -> 4 [label=140.0];
    27 -> 14 [label=137.0];
    28 -> 11 [label=153.5];
    28 -> 27 [label=68.0];
    29 -> 8 [label=192.0];
    29 -> 13 [label=152.0];
    30 -> 10 [label=117.5];
    30 -> 24 [label=59.0];
}
//2832.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=45.5];
    16 -> 22 [label=50.5];
    17 -> 9 [label=88.5];
    17 -> 18 [label=115.5];
    18 -> 7 [label=154.0];
    18 -> 19 [label=71.5];
    19 -> 6 [label=162.5];
    19 -> 21 [label=59.5];
    20 -> 4 [label=174.0];
    20 -> 12 [label=196.0];
    21 -> 2 [label=162.0];
    21 -> 20 [label=67.0];
    22 -> 17 [label=28.5];
    22 -> 23 [label=42.0];
    23 -> 3 [label=112.0];
    23 -> 26 [label=88.0];
    24 -> 5 [label=173.5];
    24 -> 29 [label=76.5];
    25 -> 1 [label=106.5];
    25 -> 10 [label=100.5];
    26 -> 28 [label=70.5];
    26 -> 30 [label=57.5];
    27 -> 15 [label=155.0];
    27 -> 25 [label=61.5];
    28 -> 11 [label=162.0];
    28 -> 14 [label=144.0];
    29 -> 8 [label=190.0];
    29 -> 13 [label=163.0];
    30 -> 24 [label=62.0];
    30 -> 27 [label=47.0];
}
//2711.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=40.5];
    16 -> 22 [label=45.5];
    17 -> 9 [label=76.5];
    17 -> 18 [label=125.0];
    18 -> 7 [label=153.5];
    18 -> 19 [label=71.0];
    19 -> 6 [label=161.5];
    19 -> 21 [label=59.5];
    20 -> 4 [label=172.0];
    20 -> 12 [label=195.0];
    21 -> 2 [label=161.0];
    21 -> 20 [label=65.5];
    22 -> 17 [label=28.0];
    22 -> 23 [label=47.5];
    23 -> 3 [label=112.5];
    23 -> 27 [label=86.5];
    24 -> 26 [label=63.0];
    24 -> 29 [label=66.0];
    25 -> 1 [label=88.5];
    25 -> 10 [label=87.5];
    26 -> 5 [label=187.5];
    26 -> 15 [label=150.5];
    27 -> 28 [label=71.0];
    27 -> 30 [label=51.0];
    28 -> 11 [label=159.0];
    28 -> 14 [label=147.0];
    29 -> 8 [label=191.0];
    29 -> 13 [label=154.0];
    30 -> 24 [label=59.0];
    30 -> 25 [label=49.0];
}
//2679.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=42.0];
    16 -> 21 [label=47.0];
    17 -> 9 [label=80.5];
    17 -> 18 [label=116.5];
    18 -> 7 [label=157.5];
    18 -> 19 [label=66.5];
    19 -> 6 [label=164.5];
    19 -> 20 [label=66.0];
    20 -> 2 [label=173.0];
    20 -> 12 [label=207.0];
    21 -> 17 [label=26.5];
    21 -> 22 [label=45.0];
    22 -> 3 [label=110.5];
    22 -> 25 [label=86.0];
    23 -> 5 [label=175.5];
    23 -> 29 [label=77.0];
    24 -> 1 [label=104.5];
    24 -> 10 [label=104.5];
    25 -> 28 [label=72.5];
    25 -> 30 [label=54.5];
    26 -> 4 [label=133.5];
    26 -> 14 [label=127.5];
    27 -> 15 [label=157.5];
    27 -> 24 [label=61.5];
    28 -> 11 [label=152.0];
    28 -> 26 [label=66.0];
    29 -> 8 [label=195.0];
    29 -> 13 [label=159.0];
    30 -> 23 [label=60.5];
    30 -> 27 [label=43.0];
}
//2673.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=42.0];
    16 -> 21 [label=47.0];
    17 -> 9 [label=80.5];
    17 -> 18 [label=116.5];
    18 -> 7 [label=157.5];
    18 -> 19 [label=66.5];
    19 -> 6 [label=164.5];
    19 -> 20 [label=66.0];
    20 -> 2 [label=173.0];
    20 -> 12 [label=207.0];
    21 -> 17 [label=26.5];
    21 -> 22 [label=45.0];
    22 -> 3 [label=110.5];
    22 -> 25 [label=86.0];
    23 -> 5 [label=175.5];
    23 -> 29 [label=77.0];
    24 -> 1 [label=104.5];
    24 -> 10 [label=104.5];
    25 -> 28 [label=72.5];
    25 -> 30 [label=54.5];
    26 -> 4 [label=133.5];
    26 -> 14 [label=127.5];
    27 -> 15 [label=157.5];
    27 -> 24 [label=61.5];
    28 -> 11 [label=152.0];
    28 -> 26 [label=66.0];
    29 -> 8 [label=195.0];
    29 -> 13 [label=159.0];
    30 -> 23 [label=60.5];
    30 -> 27 [label=43.0];
}
//2673.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=42.0];
    16 -> 21 [label=47.0];
    17 -> 9 [label=80.5];
    17 -> 18 [label=116.5];
    18 -> 7 [label=157.5];
    18 -> 19 [label=66.5];
    19 -> 6 [label=164.5];
    19 -> 20 [label=66.0];
    20 -> 2 [label=173.0];
    20 -> 12 [label=207.0];
    21 -> 17 [label=26.5];
    21 -> 22 [label=45.0];
    22 -> 3 [label=110.5];
    22 -> 25 [label=86.0];
    23 -> 5 [label=175.5];
    23 -> 29 [label=77.0];
    24 -> 1 [label=104.5];
    24 -> 10 [label=104.5];
    25 -> 28 [label=72.5];
    25 -> 30 [label=54.5];
    26 -> 4 [label=133.5];
    26 -> 14 [label=127.5];
    27 -> 15 [label=157.5];
    27 -> 24 [label=61.5];
    28 -> 11 [label=152.0];
    28 -> 26 [label=66.0];
    29 -> 8 [label=195.0];
    29 -> 13 [label=159.0];
    30 -> 23 [label=60.5];
    30 -> 27 [label=43.0];
}
//2673.0
digraph G {
	rankdir = LR;	node [ shape = none];

    0 [label=Alentus];
    1 [label=Amblypygid];
    2 [label=Americhernus];
    3 [label=Aportus];
    4 [label=Artemia];
    5 [label=Centruoides];
    6 [label=Gea];
    7 [label=Hadrurus];
    8 [label=Hypochilus];
    9 [label=Limulus];
    10 [label=Mastigoproctus];
    11 [label=Pauroctonus];
    12 [label=Rhiphicephalus];
    13 [label=Thelichoris];
    14 [label=Thermobius];
    15 [label=Vonones];
    16 [label=HTU16];
    17 [label=HTU17];
    18 [label=HTU18];
    19 [label=HTU19];
    20 [label=HTU20];
    21 [label=HTU21];
    22 [label=HTU22];
    23 [label=HTU23];
    24 [label=HTU24];
    25 [label=HTU25];
    26 [label=HTU26];
    27 [label=HTU27];
    28 [label=HTU28];
    29 [label=HTU29];
    30 [label=HTU30];
    16 -> 0 [label=49.0];
    16 -> 21 [label=56.0];
    17 -> 9 [label=88.0];
    17 -> 18 [label=113.5];
    18 -> 7 [label=158.0];
    18 -> 19 [label=65.5];
    19 -> 6 [label=164.5];
    19 -> 20 [label=64.5];
    20 -> 2 [label=174.0];
    20 -> 12 [label=209.0];
    21 -> 17 [label=34.5];
    21 -> 22 [label=46.0];
    22 -> 3 [label=118.0];
    22 -> 26 [label=85.0];
    23 -> 25 [label=66.0];
    23 -> 29 [label=70.0];
    24 -> 1 [label=90.0];
    24 -> 10 [label=85.0];
    25 -> 5 [label=180.0];
    25 -> 15 [label=149.0];
    26 -> 28 [label=62.5];
    26 -> 30 [label=47.5];
    27 -> 4 [label=136.5];
    27 -> 14 [label=123.5];
    28 -> 11 [label=140.0];
    28 -> 27 [label=70.0];
    29 -> 8 [label=190.0];
    29 -> 13 [label=147.0];
    30 -> 23 [label=61.0];
    30 -> 24 [label=52.0];
}
//2662.0
