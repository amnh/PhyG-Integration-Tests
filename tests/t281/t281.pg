--build(graph:hardwired), inputting nucleotide file
set(seed:1634561640)
set(graphtype:hardwired)
read(nucleotide:"chel-16.fas")
read(nucleotide:"chel-16_16Schopped.fas")
build(distance, rdwag, block, eun, displaytrees:10, graph)
report("t281.csv", data, overwrite)
report("t281.tre", newick, overwrite)
report("t281.dot", dotpdf, overwrite)