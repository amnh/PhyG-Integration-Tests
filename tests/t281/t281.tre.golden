(Alentus:101.0,((((Vonones:145.0,(Limulus:165.0,(Pauroctonus:1009.5)#HTU38:1009.5)HTU17:69.5)HTU22:43.0,(Amblypygid:1011.5)#HTU35:1011.5)HTU18:42.0,((Rhiphicephalus:162.0,(Artemia:131.5,Thermobius:130.5)HTU24:64.0)HTU23:64.5,(Americhernus:1055.0)#HTU36:1055.0)HTU21:65.0)HTU19:36.0,(Aportus:188.0,((((Thelichoris:171.0,(Hypochilus:1103.0)#HTU37:1103.0)HTU20:39.0,(((Gea:1144.0,#HTU37:42.0)HTU33:0.0,#HTU36:81.0)HTU29:1090.5)#HTU39:1090.5)HTU26:55.0,#HTU38:1023.5)HTU28:52.0,((Centruoides:172.0,Hadrurus:142.0)HTU32:69.5,((Mastigoproctus:100.5,#HTU35:988.5)HTU31:64.5,#HTU39:1091.0)HTU34:34.0)HTU40:32.5)HTU30:34.5)HTU27:48.0)HTU25:105.0)HTU16[3578.0];
(Alentus:94.0,(((Amblypygid:130.5,(Vonones:146.0,(Limulus:164.0,Pauroctonus:189.0)HTU17:70.0)HTU22:46.5)HTU18:49.5,(Americhernus:154.0,(Rhiphicephalus:160.5,(Artemia:129.0,Thermobius:131.0)HTU24:64.5)HTU23:64.0)HTU21:59.0)HTU19:43.0,(Aportus:177.5,((Hypochilus:181.0,Thelichoris:150.0)HTU20:74.0,((Centruoides:177.0,Hadrurus:153.0)HTU28:64.0,(Gea:193.5,Mastigoproctus:157.5)HTU29:42.5)HTU30:45.5)HTU27:46.0)HTU26:54.0)HTU25:100.0)HTU16[2883.0];
(Alentus:90.0,(((Amblypygid:123.0,(Limulus:164.0,Vonones:162.0)HTU20:68.5)HTU17:33.5,(Americhernus:162.5,(Rhiphicephalus:168.0,(Artemia:130.5,Thermobius:131.5)HTU22:63.0)HTU21:70.0)HTU19:64.5)HTU18:44.5,(Aportus:172.5,((Pauroctonus:163.0,Thelichoris:174.0)HTU25:61.0,((Centruoides:163.0,Hadrurus:147.0)HTU27:73.5,(Mastigoproctus:132.0,(Gea:160.0,Hypochilus:188.0)HTU28:82.0)HTU29:44.5)HTU30:45.5)HTU26:45.5)HTU24:43.5)HTU23:97.0)HTU16[2852.0];
(Alentus:91.0,(((Amblypygid:130.0,(Vonones:146.5,(Limulus:157.0,Pauroctonus:196.0)HTU17:65.5)HTU20:43.0)HTU18:35.5,(Rhiphicephalus:136.0,(Artemia:112.5,Thermobius:114.5)HTU22:82.5)HTU21:68.0)HTU19:43.0,(Aportus:161.0,(Thelichoris:171.5,((Centruoides:169.0,Hadrurus:147.0)HTU27:72.0,(Mastigoproctus:131.5,(Americhernus:162.5,(Gea:164.0,Hypochilus:209.0)HTU28:57.0)HTU25:81.0)HTU29:44.0)HTU30:58.0)HTU26:58.0)HTU24:54.5)HTU23:92.0)HTU16[2830.0];
(Alentus:91.0,(((Amblypygid:130.0,(Vonones:146.5,(Limulus:157.0,Pauroctonus:196.0)HTU17:65.5)HTU20:43.0)HTU18:35.5,(Rhiphicephalus:136.0,(Artemia:112.5,Thermobius:114.5)HTU22:82.5)HTU21:68.0)HTU19:43.0,(Aportus:161.0,(Thelichoris:171.5,((Centruoides:169.0,Hadrurus:147.0)HTU27:72.0,(Mastigoproctus:131.5,(Americhernus:162.5,(Gea:164.0,Hypochilus:209.0)HTU28:57.0)HTU25:81.0)HTU29:44.0)HTU30:58.0)HTU26:58.0)HTU24:54.5)HTU23:92.0)HTU16[2830.0];
(Alentus:89.0,(((Amblypygid:131.5,(Limulus:157.5,Vonones:155.5)HTU19:66.0)HTU17:32.5,(Rhiphicephalus:139.0,(Artemia:115.0,Thermobius:121.0)HTU21:80.5)HTU20:61.5)HTU18:40.0,(Aportus:164.0,((Pauroctonus:156.5,Thelichoris:183.5)HTU24:63.0,((Centruoides:170.0,Hadrurus:149.0)HTU27:70.5,(Mastigoproctus:140.0,(Americhernus:158.0,(Gea:160.0,Hypochilus:208.0)HTU28:59.5)HTU25:75.5)HTU29:48.5)HTU30:48.0)HTU26:49.5)HTU23:51.0)HTU22:95.0)HTU16[2803.0];
(Alentus:93.0,(((Amblypygid:133.5,(Vonones:145.5,(Limulus:155.0,Pauroctonus:191.0)HTU17:63.5)HTU20:41.0)HTU18:36.5,(Rhiphicephalus:138.5,(Artemia:113.5,Thermobius:113.5)HTU22:80.0)HTU21:64.5)HTU19:35.0,(Aportus:175.5,((Thelichoris:154.0,(Americhernus:146.5,(Gea:151.5,Hypochilus:194.5)HTU29:61.5)HTU26:52.0)HTU24:75.0,(Mastigoproctus:121.5,(Centruoides:163.0,Hadrurus:147.0)HTU28:75.0)HTU30:53.0)HTU27:44.5)HTU25:37.0)HTU23:101.0)HTU16[2791.0];
(Alentus:93.0,(((Amblypygid:133.5,(Vonones:145.5,(Limulus:155.0,Pauroctonus:191.0)HTU17:63.5)HTU20:41.0)HTU18:36.5,(Rhiphicephalus:138.5,(Artemia:113.5,Thermobius:113.5)HTU22:80.0)HTU21:64.5)HTU19:35.0,(Aportus:175.5,((Thelichoris:154.0,(Americhernus:146.5,(Gea:151.5,Hypochilus:194.5)HTU29:61.5)HTU26:52.0)HTU24:75.0,(Mastigoproctus:121.5,(Centruoides:163.0,Hadrurus:147.0)HTU28:75.0)HTU30:53.0)HTU27:44.5)HTU25:37.0)HTU23:101.0)HTU16[2791.0];
(Alentus:93.0,(((Amblypygid:128.0,(Limulus:162.5,Vonones:150.5)HTU20:65.0)HTU17:31.0,(Rhiphicephalus:138.0,(Artemia:116.5,Thermobius:115.5)HTU22:80.5)HTU21:66.5)HTU18:36.5,(Aportus:168.0,((Pauroctonus:152.5,((Hypochilus:184.5,Thelichoris:164.5)HTU19:55.5,(Americhernus:156.0,Gea:156.0)HTU27:56.0)HTU24:61.0)HTU26:57.5,(Mastigoproctus:129.0,(Centruoides:165.5,Hadrurus:150.5)HTU29:73.0)HTU30:59.0)HTU28:38.5)HTU25:41.5)HTU23:101.0)HTU16[2758.0];
(Alentus:84.0,(((Vonones:127.5,(Limulus:141.0,Pauroctonus:187.0)HTU17:67.0)HTU19:36.5,(Rhiphicephalus:129.5,(Artemia:114.0,Thermobius:117.0)HTU21:90.0)HTU20:49.5)HTU18:38.0,(Aportus:167.5,((Thelichoris:158.0,(Americhernus:150.0,(Gea:155.0,Hypochilus:201.0)HTU29:56.5)HTU25:51.0)HTU23:79.0,((Amblypygid:105.0,Mastigoproctus:97.0)HTU27:58.5,(Centruoides:165.0,Hadrurus:152.0)HTU28:68.5)HTU30:52.5)HTU26:48.5)HTU24:56.0)HTU22:93.0)HTU16[2695.0];
(Alentus:82.5,(((Limulus:151.5,Vonones:146.5)HTU18:50.0,(Rhiphicephalus:129.0,(Artemia:115.0,Thermobius:120.0)HTU20:86.0)HTU19:55.5)HTU17:31.0,(Aportus:166.5,((Pauroctonus:160.5,(Thelichoris:159.0,(Americhernus:157.5,(Gea:167.0,Hypochilus:203.0)HTU29:58.0)HTU25:51.5)HTU22:66.5)HTU24:55.5,((Amblypygid:101.0,Mastigoproctus:101.0)HTU27:58.5,(Centruoides:168.0,Hadrurus:155.0)HTU28:75.0)HTU30:49.0)HTU26:49.0)HTU23:58.5)HTU21:92.5)HTU16[2668.0];
